src/helper_msgcoding.py

```diff
@@ -140,22 +142,26 @@ def decodeSimple(self, data):
         self.body = body
 
 if __name__ == '__main__':
```

Comment:
Can you make this into a unit test? https://docs.python.org/2/library/unittest.html